tasks/agent.py

```diff
@@ -127,6 +129,7 @@ def build(
     bundle=None,
     bundle_ebpf=False,
     agent_bin=None,
+    platform=None,  # noqa: U100. Used by the run_on_devcontainer decorator
```

Comment:
If I understand correctly if the platform argument is provided then the task will actually be ran in the devcontainer, what do you think of renaming this to `devc_platform` (or anything mentioning devcontainer really) ?
I feel like it would make it clearer for users, and it would limit the risks of providing the argument thinking it's just a generic `platform` arg

Reply:
What about `--run-on` so it opens the door to more stuff in the future, not just devcontainer?

Reply:
I think it's better yep
Can you add an explicit help message for this arg ? (possibly listing allowed values)